.travis.yml

```diff
@@ -124,35 +107,37 @@ jobs:
           - ws_amd
           - dockerconfig
 
-    - <<: *deploy
-      name: Deploy snapshot quay AMD
-      arch: amd64
-      if: (branch != master) AND (tag IS blank)
+    - <<: *deploy_amd
```

Comment:
We're no longer publishing to Quay for releases, only development builds.

Reply:
Changed condition to "if: (branch != master) AND (tag IS blank)"

Reply:
Suggest renaming it to "Deploy snapshot quay AMD"

Please also do the name/if changes to the ARM job.

Reply:
Thanks for the hint! Done